frontend/src/libs/components/input/styles.module.scss

```diff
@@ -18,6 +18,20 @@
   outline: none;
 }
 
+.search-input {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 36px;
+  padding: 6px 8px 6px 32px;
+  color: var(--gray-600);
+  font-size: 14px;
+  line-height: 24px;
+  background-color: var(--white);
+  border: 2px solid var(--black);
+  outline: none;
+}
```

Comment:
The only this that was changed from `.input` class is a padding - why do we need to duplicate the styles?